README.md

```diff
@@ -2,264 +2,65 @@
--  a cluster administrator can build an abstract registry of immutable images (referred to as "Golden Images") which can be cloned and later consumed by Kubevirt, or
--  an ad-hoc user (granted access) can import a VM image into their own namespace and feed this image directly to Kubevirt, bypassing the cloning step.
-
-
-For an in depth look at the system and workflow, see the [Design](/doc/design.md#design) documentation.
-
-### Versions
-
-CDI follows the common semantic version scheme defined at semver.org in the "vMajor.Minor.Patch" pattern.  These are defined as:
-
-- Major: API or other changes that will break existing CDI deployments.  These changes are expected to require users to alter the way they interact with CDI.  Major versions are released at the end a development cycle (every 2 weeks) in leiu of a Minor version.
-
-- Minor: Backwards compatible changes within the current Major version.  These changes represent the products of a 2 week developement cycle and contain bug fixes and new features.  By releasing merged work at the end of the cycle, users are able to closely track the project's progress and report issues or bugs soon after they are introduced.  At the same time, it should be easy to roll back to the previous Minor version if the release blocks the user's workflow.
-
-- Patch: Mid development cycle critical bug fix. In the case that a Minor release has a bug that must be fixed for users before the next release cycle, a Patch may be published.  Patches should be small in scope and only alter / introduce code related to the bug fix.
-
-See [releases.md](/doc/releases.md) for more information on versioning.
-
-### Data Format
-
-The importer is capable of performing certain functions that streamline its use with Kubevirt.  It automatically decompresses **gzip** and **xz** files, and un-tar's **tar** archives. Also, **qcow2** images are converted into a raw image files needed by Kubevirt, resulting in the final file being a simple _.img_ file.
-
-Supported file formats are:
-
-- .tar
-- .gz
-- .xz
-- .img
-- .iso
-- .qcow2
-
-## Deploying CDI
-
-### Assumptions
-- A running Kubernetes cluster with roles and role bindings implementing security necesary for the CDI controller to watch PVCs and pods across all namespaces.
-- A storage class and provisioner.
-- An HTTP or S3 file server hosting VM images
-
-- An optional "golden" namespace acting as the image registry. The `default` namespace is fine for tire kicking.
-
-### Download CDI
-
-`$ go get kubevirt.io/containerized-data-importer`
-
-*Or*
-
-`$ git clone https://github.com/kubevirt/containerized-data-importer.git $GOPATH/src/kubevirt.io/containerized-data-importer`
+Deploying the CDI controller is straight forward. In this document the _default_ namespace is used, but in a production setup a [protected namespace](#protecting-the-golden-image-namespace) that is inaccessible to regular users should be used instead.
 
-*Or download only the yamls:*
+  ```
```

Comment:
i think that starting from 1.5.2, we want users to deploy from the operator yaml (as it also bundles the required scc and the configmap)

Reply:
Agreed, we should only support deploying with the operator, so the main example should use it as well.